docrepo.bib

```diff
@@ -148,23 +146,20 @@ @MISC{2018ivoa.spec.0529H
       adsnote = {Provided by the SAO/NASA Astrophysics Data System}
 }
 
-@MISC{2018ivoa.spec.0527M,
-       author = {{Martinez}, Andrea Preite and {Louys}, Mireille and {Cecconi}, Baptiste and
-         {Derriere}, Sebastien and {Ochsenbein}, Fran{\c{c}}ois and
-         {IVOA Semantic Working Group}},
+@MISC{2018ivoa.spec.0527P,
```

Comment:
Interesting that the bibcode changed when the author name was corrected? This will cause problems for any document referencing it